spec/graphql/bamboohr_spec.rb

```diff
@@ -0,0 +1,60 @@
+require 'rails_helper'
+require 'bamboozled'
+
+EMPLOYEESDATA = [
+  { "id" => "4", "displayName" => "Charlotte Abbott", "hireDate" => Time.zone.today.to_formatted_s(:iso8601),
+    "birthday" => "07-29", "isPhotoUploaded" => "true",
+    :photoUrl => "http://blahbambooapitestblah.bamboohr.com/employees/photos/?h=e27ddc876e2b95ca0ac80ad62967506a" },
+  { "id" => "5", "displayName" => "Ashley Adams", "hireDate" => "2020-12-18",
+    "birthday" => "05-25", "isPhotoUploaded" => "true",
+    :photoUrl => "http://blahbambooapitestblah.bamboohr.com/employees/photos/?h=efbf026fde3168eb96a3447b040f5f91" },
+  { "id" => "6", "displayName" => "Christina Agluinda", "hireDate" => "2020-08-13",
+    "birthday" => Time.zone.now.strftime("%m-%d"), "isPhotoUploaded" => "true",
+    :photoUrl => "http://blahbambooapitestblah.bamboohr.com/employees/photos/?h=206a1f7e6807f2dac63a7611dfce8453" }
+].freeze
+
+TIMEOFFDATA = [
+  { "id" => 1501, "type" => "timeOff", "employeeId" => 5, "name" => "Ashley Adams",
+    "start" => Time.zone.today.to_formatted_s(:iso8601), "end" => Time.zone.tomorrow.to_formatted_s(:iso8601) }
+].freeze
+
+describe Helios2Schema do
```

Comment:
Nit: we are describing `Clients::CreateEmployees` in our spec.

Reply:
This should be fixed now!